packages/components/src/tooltip/test/index.js

```diff
@@ -1,373 +1,323 @@
 /**
  * External dependencies
  */
-import { render, screen, act, waitFor } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import Tooltip from '../';
+import Button from '../../button';
+
 /**
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
```

Comment:
This looks like an internal dependency, no?

I realize this was here before, but perhaps we could address it while we're here?

Reply:
Yes, good catch!